### DISCUSSION TITLE
Hotkey

### DISCUSSION THREAD

**Discussion by yip623** [dail8859/NotepadNext#114] [Ideas]
Nice to have cross-platform Notepad++
If possible to have the same hotkey with Notepad++, it would be a perfect replacement.

**Comment by dail8859** [comment]
Hi @yip623 

My long-term plan as far as shortcuts:

- Try to match Notepad++ when possible and it makes sense.
- If there is good reason to make a shortcut different than Notepad++ then I'm fine with that.
- If things start to become too different than Notepad++, I think it would be beneficial to provide the user some built in option to restore Notepad++ shortcuts.
- Make shortcuts configurable
- Possibly add a feature to import Notepad++ settings including shortcuts
